### DISCUSSION TITLE
Isoform Expression of Zero in Condition

### DISCUSSION THREAD

**Discussion by erichpeterson** [kvittingseerup/IsoformSwitchAnalyzeR#117] [Q&A]
Hi,
I was wondering if IsofromSwitchAnalyzeR only looks at isoforms that are expressed in all of the conditions under study. It seems to me that this would be necessary to do DTU analyses.

Thanks

**Comment by kvittingseerup** [comment]
Thanks for bringing this to my attention. I found a mistake that causes the filtering to work on the average value across all data (instead of the two individual condition averages). This will be fixed in the next update.

Cheers
Kristoffer
